src/pat/ajax/tests.js

```diff
@@ -1,6 +1,13 @@
 define(["pat-ajax"], function(pattern) {
     var $lab;
 
+    // Monkey patch Date.now so that we dont get current time mismatches while testing
+    var originalDatenow = Date.now;
+    Date.now = function() {
+        originalDatenow();
+        return "123"; // dummy data instead of current time
+    };
+
```

Comment:
Please use the Jasmine `beforeEach` method here, and also in `afterEach`. In `afterEach` you restore the original function, i.e. ` Date.now = originalDatenow;`.

BTW, why do you call `originalDatenow` in your monkey patch?